services/frontend-v3/src/components/careerInterests/CareerInterests.jsx

```diff
@@ -1,14 +1,56 @@
 import React, { Component } from "react";
 import CareerInterestsView from "./CareerInterestsView";
+import { FormattedMessage, injectIntl } from "react-intl";
 
-class CareerInterests extends Component {
-  render() {
-    const { data } = this.props;
+function CareerInterests(props) {
+  const getCareerInterestsInfo = dataSource => {
+    const data = dataSource.data;
+    const locale = localStorage.getItem("lang");
 
-    return (
-      <CareerInterestsView data={data} locale={localStorage.getItem("lang")} />
-    );
-  }
+    const interestedInRemote = {
+      icon: "mail",
+      title: <FormattedMessage id="profile.interested.in.remote" />,
+      description:
+        data.interestedInRemote === true ? (
+          <FormattedMessage id="profile.yes" />
+        ) : (
+          <FormattedMessage id="profile.no" />
+        )
+    };
+    const lookingForNewJob = {
+      icon: "mail",
+      title: <FormattedMessage id="profile.looking.for.new.job" />,
+      description: (data.lookingForNewJob &&
+        data.lookingForNewJob.description[locale]) || (
+        <FormattedMessage id="profile.not.specified" />
+      )
+    };
+
+    return [interestedInRemote, lookingForNewJob];
+  };
+
+  const getRelocationLocationsInfo = dataSource => {
+    const data = dataSource.data;
+    const locale = localStorage.getItem("lang");
+
+    const relocationLocationsInfo = [];
+    if (data.relocationLocations) {
+      data.relocationLocations.forEach(locationElement =>
+        relocationLocationsInfo.push(locationElement.description[locale])
+      );
+    }
+
+    return [...relocationLocationsInfo];
+  };
+
+  return (
+    <CareerInterestsView
+      data={props.data}
+      locale={localStorage.getItem("lang")}
+      info={getCareerInterestsInfo(props)}
+      relocationLocationsInfo={getRelocationLocationsInfo(props)}
+    />
+  );
 }
 
-export default CareerInterests;
+export default injectIntl(CareerInterests);
```

Comment:
Are you adding injectIntl when you aren't using props.intl?

Reply:
Yes.. unfortunately I was. 😞  Fixed it tho!